### PR TITLE
Some material UI clean up and introduce new button style

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

- We were not really using material ui for anything important in the side navigation that we couldn't easily do with plain html, this PR removes those items
- Since we are slowing down shared feed development, we don't need different "types" of shared feeds right now, so I removed the "Collaborating" header wording and removed the collapse functionality, since its now just a single list and expand/collapse is not needed.
- Added a new Main action button style for side navigations. its used first here for shared feeds, but will eventually be used for upcoming Articles work
- updated some setting of css classes from the join array style to us the classnames bind approach

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

manual verification, running integration tests

## Things to pay attention to during code review

no significant functionality or styles changes

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by brianfleming** [comment]
I think the integration test failure are the typical flaky ones?

File: src/app/components/drawer/Projects/FeedsComponent.js [hunk 1]
```diff
@@ -87,91 +62,80 @@ const FeedsComponent = ({
...
-                description="Collaborating is a label to describe the type of shared feeds listed. In this case it means that they are groups of workspaces working together."
-                values={{ feedsLength: feeds.length }}
+      <Can permissions={team.permissions} permission="create Feed">
+        <div className={styles.listMainAction}>
+          <ButtonMain
+            className="projects-list__add-feed"
+            label={
+              <FormattedHTMLMessage
+                id="projectsComponent.newSharedFeed"
+                defaultMessage="New Shared Feed"
```
Comment (amoedoamorim) [inline src/app/components/drawer/Projects/FeedsComponent.js:77]: Is this meant to be FormattedHMTL?

**Review by amoedoamorim — APPROVED** [review]
LGTM

**Comment by amoedoamorim** [comment]
> I think the integration test failure are the typical flaky ones?

Yeah, those are the classic "Flaky Four" and a seemingly unrelated one

**Pull request merged**